docs/tutorials/extending_admin_ui/3b_adding_edit_button_content_list.md

```diff
@@ -0,0 +1,229 @@
+# Step 3b - Adding Edit button to content list
+
+In this step you will add an edit button to the content list. The following tutorial is based on the [Step 2 - Creating a content list](2_creating_a_content_list.md).
+
+## Modify the controller
+
+Introduce changes to `src/Controller/AllContentListController.php`, so that it allows content editing from the content list.
+
+First, add the new `FormFactory $formFactory` parameter and the `$this->formFactory = $formFactory;` argument to the `__construct` function:
+
+```php hl_lines="1 5"
+public function __construct(SearchService $searchService, ContentTypeService $contentTypeService, FormFactory $formFactory)
+{
+    $this->searchService = $searchService;
+    $this->contentTypeService = $contentTypeService;
+    $this->formFactory = $formFactory;
+}
+```
+
+Next, provide a new use statement for `FormFactory` parameter:
+
+```php
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
+```
+
+After the lines setting the `$paginator` parameters, add the following code line:
```

Comment:
it dosent have to be after those lines, I would also explain what this do - creating underlining form for handling request for content editing.